README.md

```diff
@@ -1,59 +1,100 @@
 # Freifunk Berlin Firmware
 https://wiki.freifunk.net/Berlin:Firmware
 
-This is the build-system for the Firmware of Freifunk Berlin.
+This is the build system for the firmware of Freifunk Berlin.
 The firmware is based on vanilla [OpenWrt](https://openwrt.org/start) with some modifications (to fix
 broken stuff in OpenWrt itself or for example LuCI) and additional default packages/configuration settings.
 
 ## Contact / More information
 
-More user relevant information about the firmware are on the wiki page at: https://wiki.freifunk.net/Berlin:Firmware. There you can also find the
-* [ReleaseNotes](https://wiki.freifunk.net/Berlin:Firmware/v1.0.2)
+More user-relevant information on the firmware can be found on the wiki page at: https://wiki.freifunk.net/Berlin:Firmware. There you can also find the
+* [Release Notes](https://wiki.freifunk.net/Berlin:Firmware/v1.0.2)
 * a tutorial ([en](https://wiki.freifunk.net/Berlin:Firmware:En:Howto) / [de](https://wiki.freifunk.net/Berlin:Firmware:Howto)) on router configuration
 
-For questions write a mail to <berlin@berlin.freifunk.net>, open a discussion with 
+If you have any questions, send an email to <berlin@berlin.freifunk.net>, open a discussion with 
 @freifunk-berlin/firmware or come to our weekly meetings.
-If you find bugs please report them at: https://github.com/freifunk-berlin/firmware/issues
+If you find bugs, please report them at: https://github.com/freifunk-berlin/firmware/issues
 
 ## Development
 
 ### Info
 
 For the Berlin Freifunk firmware we use vanilla OpenWrt with additional patches and packages.
-The Makefile automates firmware creation (applies patches, integrates custom packages and uses
-the Imagebuilder of OpenWrt).
+The Makefile automates the build of the firmware (applies patches, integrates custom packages and uses the
+the ImageBuilder of OpenWrt).
 
 The idea is to download OpenWrt via git, patch it with all patches in the `patches/` folder
-and configure it according to the configuration snippets in `configs/`. Then use the OpenWrt
-buildsystem to build all packages and the Imagebuilder. The packages to assemble into the final
-image for the router are taken from the imageflavors defined in `packagelist/`.
+patch it and configure it according to the configuration snippets in `configs/`. Then use the OpenWrt
```

Comment:
"... patch it with all patches in the `patches/` folder patch it and configure it ..."